shared.props

```diff
@@ -0,0 +1,37 @@
+<Project>
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\Steeltoe.Debug.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\Steeltoe.Release.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\stylecop.json" Visible="false" />
```

Comment:
I think this file previously also had `<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>` attached to it, because the default value prevented incremental builds from working (for example, if you made a change in Management.EndpointCore, all projects it depends on would rebuild first even if nothing changed) ~ unless something has changed in the build tooling, the loss of this value could result in wasted compute time

Reply:
As documented [here](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/EnableConfiguration.md), there's no need to copy `stylecop.json` to the output directory. This makes sense to me, because project items of type `AdditionalFile` were specifically designed to be consumable by roslyn analyzers. Roslyn analyzers only execute at compile time, so there's no point in copying them to the output directory to be consumed at runtime.

Looking at history, it used to be `<CopyToOutputDirectory>Always</CopyToOutputDirectory>`, which basically means the parent project changes always, resulting in an (unneeded) rebuild.